From ad10c51025d4e5d21f326865d10147e8e81719ec Mon Sep 17 00:00:00 2001
From: Ran Shalit <rshalit@habana.ai>
Date: Sun, 16 Feb 2025 11:20:15 +0200
Subject: [PATCH] [SW-211822] include: add config for empty weak func

define CONFIG_EMPTY_WEAK_FUNCTIONS, so that by default
all weak function are enabled and implemented
(with a warning message)

Change-Id: I7b94f8b5d5d567c95fd4b244ce682fe4209b4efb
Signed-off-by: Ran Shalit <rshalit@habana.ai>
---

diff --git a/include/baremetal.h b/include/baremetal.h
index f6f07c4..97dfd8d 100644
--- a/include/baremetal.h
+++ b/include/baremetal.h
@@ -47,7 +47,7 @@
 #define FIT_IMG_SPSRAM_BASE		(FIT_IMG_SPSRAM_END - \
 					 FIT_IMG_SPSRAM_MAX_SIZE)
 #define FIT_CONF_NAME			"recovery-conf"
-
+#define CONFIG_EMPTY_WEAK_FUNCTIONS
 
 #ifndef __ASSEMBLY__
 
